Assets/Scripts/SS3D/Systems/Screens/GameScreens.cs

```diff
@@ -0,0 +1,78 @@
+﻿using JetBrains.Annotations;
+using SS3D.Core;
+using SS3D.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace SS3D.Systems.Screens
+{
+    public static class GameScreens
```

Comment:
For the sake of clarity, it'd be nice if you could add the [Server] [Client] [ServerOrClient] attributes on relevant methods. That way we understand better what execute what and it limits error due to server or client calling methods they're not supposed to.